server/embed/etcd.go

```diff
@@ -411,6 +411,16 @@ func (e *Etcd) Close() {
 		close(e.stopc)
 	})
 
+	for i := range e.Clients {
+		if e.Clients[i] != nil {
```

Comment:
The Clients here is about net listener. If we close it at the beginning, all the connections will be closed, and it seems like we are unable to drain all the inflight requests gracefully. The `stopServers` function is used to gracefully shutdown.

Reply:
> The Clients here is about net listener. If we close it at the beginning, all the connections will be closed,

It's intentional.  
- We should stop accepting any new connections immediately.
- It won't close the already established connections

Reply:
The description has already been updated. Where did you get the description?

Reply:
In the pull request's description

Reply:
Oh, updated. thx